libraries/Microsoft.Bot.Builder.Core/BotContext.cs

```diff
@@ -74,6 +74,14 @@ public bool Responded
             }
         }
 
+        public async Task SendActivity(params IMessageActivity[] messagesToSend)
```

Comment:
this should not be `IMessageActivity` - what if the user wants to send a `Typing` activity? A conversation activity? Or a <insert other activity type here> Activity?